README.md

```diff
@@ -1,64 +1,41 @@
-IBM_DB_SA
+SQLAlchemy-IBMi
 =========
 
-The IBM_DB_SA adapter provides the Python/SQLAlchemy interface to IBM Data Servers.
+The SQLAlchemy-IBMi adapter provides the Python/SQLAlchemy interface to IBM i db2.
 
 Version
 --------
-0.3.5 (2019/05/30)
+0.1 (2019/11/26)
 
 Prerequisites
 --------------
-1. Install Python 2.7 or newer versions except python 3.3 or Jython 2.5.x .
-2. SQLAlchemy 0.7.3 or above.
-3. IBM_DB driver and IBM_DB_DBI wrapper 1.0.1 or higher.
+1. Install Python 2.7 or newer versions except python 3.3
```

Comment:
```suggestion
1. Install Python 3.5 or newer
```

We're not going to support Python 2 and Python 3 < 3.5 is no longer in support, so no reason to support those either.